### ISSUE TITLE
Cannot create more than 10 groups

### ISSUE THREAD

**Issue by takyon77** [goharbor/harbor#15958]
One of our Harbor clusters has 25 LDAP groups defined. Once I did the upgrade 2.3.0 -> 2.4.0, after I created 10 groups, any attempt to create a new group (through the GUI or the API) ends with an error "HTTP Error 409: Conflict".

**Versions:**
- harbor version: 2.4.0
- docker engine version: docker-ce-20.10
- docker-compose version: docker-compose-1.18.0

**Comment by AllForNothing** [comment]
@takyon77 Are you trying to add an existing LDAP group?
Maybe, you are facing the same issue of #15906

**Comment by takyon77** [comment]
@AllForNothing I just try to create my 25 LDAP groups in the same way I did on harbor 2.3.0. On a fresh install of Harbor 2.4.0 (so with no groups at all), after I created 10 of them, all others are failing. If I delete one of the 10 groups, then I manage to create one more. So the group creation seems to work only when there's 10 groups or less.
Issue #15906 is only affecting the display of groups correct? TO make sure I tried to list all existing groups and there's indeed only 10 of them. I can see the 25 groups by doing the same operation on Harbor 2.3.0.
